README.md

```diff
@@ -6,33 +6,42 @@
 
 To run the chatbot, then it is necessary to have `Node` installed on your system.
 
-The project has only been tested on recent versions of Node, and hence it is recommened to use Node `>= 8.0`.
+The project has been tested on multiple versions of Node, and it is recommened to use Node versions between `8.0` and `10.13`.
 
 ### Typescript
 
 Furthermore typescript is required to compile the project to Node compatible JavaScript code.
 
 To install TypeScript run the command `npm install -g typescript` or `yarn global add typescript`
 
+### Yarn
+
+In case you want to use yarn make sure you have it installed by running the command `npm install --global yarn`
+
 ## Build project
 
 The PolloPollo chat-bot project is written in TypeScript and hence compilation is required in order to run the project. Therefore, to run the bot, please use the following steps:
 
-1. Run the command `tsc` in the root of the project
-2. Run the command `yarn start`/`npm run start` to run the bot
+1. Make a file called dbData.json in the root of the project containing mySQL credentials 
```

Comment:
Could you maybe give an inline example of what this file should look like?

e.g.
```json
{
  "thisIsHow": "you do"
}
````